### PR TITLE
fixed startup.asm to stop all cores except one

### PR DESCRIPTION
**Pull request by andreaslindner**
Hi Daniel,

I couldn't get your code to run properly, probably the bootcode.bin provided by the Raspberry Pi foundation changed or something. It seems like that all four cores run and start executing the code, using the same stack pointer and everything. I had multiple times the same four letters in the output and then a lot of them were skipped.

Anyways after adding these lines, the output looks smooth at least. I'll move on with testing JTAG now.

Best,
  Andreas

### CODE REVIEW

File: startup.asm [hunk 1]
```diff
@@ -28,6 +28,10 @@
 
 .globl _start
 _start:
+	mrs x2, MPIDR_EL1
+	and x2, x2, #0xFF
+	cbnz x2, hang
+
```
Comment (daniel-k) [inline startup.asm]: Can you maybe add a short comment what this is doing and why it is needed?

**Comment by andreaslindner** [comment]
Yes, of course! I also added the required target selection to make JTAG work. It was necessary to select core0 first in OpenOCD.

**Comment by daniel-k** [comment]
Cool! Thanks for taking the time 👍 Ready to merge?

**Comment by andreaslindner** [comment]
Yes, I'm just looking into whether it's easily possible to turn on the LED. You know, just to have it nice ;). But we can make this a separate merge if I manage.

**Pull request merged**